endpoints/version.go

```diff
@@ -0,0 +1,35 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const versionEndpointValueNotSet = "not-set"
+
+// NewVersionEndpoint returns the latest git tag as the version and commit hash as the revision from which the binary was built
+func NewVersionEndpoint(version, revision string) http.HandlerFunc {
+	if version == "" {
+		version = versionEndpointValueNotSet
+	}
+	if revision == "" {
+		revision = versionEndpointValueNotSet
+	}
+
+	response, err := json.Marshal(struct {
+		Version  string `json:"version"`
+		Revision string `json:"revision"`
+	}{
+		Version:  version,
+		Revision: revision,
+	})
+	if err != nil {
+		log.Fatalf("error creating /version endpoint response: %v", err)
```

Comment:
Am I correct in assuming this will exit the application and prevent startup?

Reply:
Good catch. I copy/pasted from Prebid Server but `logrus` will exit the application. Changed to `Errorf`